CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - `Contract.Wallet.isNamiAvailable` and `Contract.Wallet.isGeroAvailable` functions ([#558](https://github.com/Plutonomicon/cardano-transaction-lib/issues/558)])
 - `Contract.Transaction.balanceTxWithOwnAddress` and `Contract.Transaction.balanceTxsWithOwnAddress` to override an `Address` used in `balanceTx` internally ([#775](https://github.com/Plutonomicon/cardano-transaction-lib/pull/775))
 - `Contract.Transaction.awaitTxConfirmedWithTimeoutSlots` waits a specified number of slots for a transaction to succeed. ([#790](https://github.com/Plutonomicon/cardano-transaction-lib/pull/790))
+- Added unzip and Chromium (optional) to Nix overlay. ([#799](https://github.com/Plutonomicon/cardano-transaction-lib/pull/799))
```

Comment:
You're actually not adding anything to the overlay (you don't need to, they're already in the package set from `nixpkgs`). You should mention the `withChromium` flag and that `unzip` is being added to the `devShell`